regression/cbmc/Pointer_byte_extract10/main.c

```diff
@@ -0,0 +1,45 @@
+void *malloc(__CPROVER_size_t);
+
+typedef union
+{
+  int a;
+  int b;
+} Union;
+
+#ifdef __GNUC__
+typedef struct
+{
+  int Count;
+  Union List[1];
+} __attribute__((packed)) Struct3;
+#else
+typedef struct
+{
+  int Count;
+  Union List[1];
+} Struct3;
+#endif
+
+int main()
+{
+  Struct3 *p = malloc (sizeof (int) + 2 * sizeof(Union));
```

Comment:
```(sizeof(Struct3) + sizeof(Union))``` because padding (think an architecture with strict alignment, 4 byte integers and 8 byte pointers)?

Reply:
Good call - done (packing is only requested on for `__GNUC__` in this test).